### ISSUE TITLE
The valid-post-logout-redirect-uris can't be set on the keycloak_openid_client

### ISSUE THREAD

**Issue by vincent-petit** [mrparkers/terraform-provider-keycloak#740]
[Context]
mrparkers/keycloak >= 3.10
Keycloak:19

[issue]
The "Valid post logout redirect URIs" present in the realm client settings can't be set using the keycloak_openid_client resources.
![image](https://user-images.githubusercontent.com/14886853/193306692-51baa077-3112-43f7-843e-c1bdffc44df0.png)
The attribute is not defined in this project, and i failed to configure it using extra_config.

I tried some syntax such as
```
  extra_config             = {
    "valid_post_logout_redirect_uris" = "+"
    "valid-post-logout-redirect-uris" = "+"
    "valid-post-logout-redirect-uris" = "[+]"
  }
```
Maybe i miss something with the extra_config syntax

The attribute type should be identical to the valid_redirect_uris and web_origins attributes

**Comment by jjojo** [comment]
Hi @vincent-petit!
I had the same issue as you and by looking at the http request when changing valid post login urls in the web GUI I found that the key we need is `post.logout.redirect.uris` and the syntax for the value is `https://yourdomain.com##https://yourOtherDomain.com` so it is a string that uses "##" as a separator. Since this worked for me it should work for you as well.

```
  extra_config = {
    "post.logout.redirect.uris" : "http://localhost:3000/*##https://anotherUrl.com",
  }
```

And if you wanna use the shorthand "+" and by that copy valid redirect URIs you can use
```
  extra_config = {
    "post.logout.redirect.uris" : "+",
  }
```

**Comment by jjojo** [comment]
#749

**Comment by mrparkers** [comment]
Fixed in #777
